lib/tasks/lead_organisation_publisher.rake

```diff
@@ -0,0 +1,22 @@
+namespace :publishing_api do
+  desc "populate publishing api with lead organisations"
+  task publish_lead_organisations: :environment do
+    Policy.find_each do |policy|
+
+      links_payload = {
+        links: {
+          "organisations" => policy.organisation_content_ids
```

Comment:
Minor thing, but [the styleguide](https://github.com/alphagov/styleguides/blob/master/ruby.md#collections) prefers `:symbols` over `"strings"`.